ros_buildfarm/templates/release/rpm/mock_config.cfg.em

```diff
@@ -41,6 +41,15 @@ config_opts['macros']['%_without_weak_deps'] = '1'
 config_opts['chroot_setup_cmd'] += ' gcc-c++ make'
 @[end if]@
 
+@[if os_name == 'rhel' and os_code_name == '9']@
+# Pin RHEL 9 to 9.1 everywhere except EPEL
+import re
+config_opts[f'{config_opts.package_manager}.conf'] = re.sub(
+    r'(?<!(epel|EPEL)-)\$releasever',
```

Comment:
Unpacking the regexp a little for review. This is a negative lookbehind for `EPEL-` (covering both casings but not mixed case), then substituting `9.1` for `$releasever`.